01/app.js

```diff
@@ -1 +1,17 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const itemsList = document.querySelector('.comments__item' && '.comments__item--newest');
```

Comment:
Tutaj prawidłowym rozwiązaniem jest `querySelector('.comments__item.comments__item--newest')`.
Twoje rozwiązanie działa tak: jeśli element po lewej stronie `&&` jest prawdziwy to zwróć ten drugi.
To akurat tutaj działa, ale logicznie nie jest poprawne, bo chcieliśmy szukać obu klas.